src/components/AboutUs/ClientTestimonial.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Padding } from 'styled-components-spacing'
+
+import { Row, Col, Grid } from '../grid'
+import { SectionTitle } from '../Typography'
+import GreyBackground from '../Common/GreyBackground'
+
+const Video = styled.iframe`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+`
+
+/**
+ * Since iframes can't set their height to auto, we're using a
+ * container with the padding bottom equal to the video width/height ratio.
+ */
+const VideoRatioContainer = styled.div`
+  position: relative;
+  height: 0;
+  padding-bottom: ${props => (props.height / props.width) * 100}%;
+`
+
+const CenteredRow = styled(Row)`
+  justify-content: center;
+`
+
+const ClientTestimonialVideo = ({ video }) => (
+  <Padding top={{ smallPhone: 3, tablet: 4 }}>
```

Comment:
This is _the_ most minor thing & there's no official defined pattern or anything anyway. But the way I've been interpreting the designs is that there seems to be a bit of a pattern of `72px` below all section titles. So just a little FYI, each of the other sub sections of this page, including your `Teams` section, has this as `Padding bottom=...` applied to the title.

That's maybe just a nice little pattern to repeat, in case this section ever gets bigger or more things get added in.

As I say, just a silly little FYI :laughing:

Reply:
makes sense, done